rules/no-ajax.js

```diff
@@ -1,24 +1,19 @@
 'use strict'
 
+const utils = require('./utils.js')
+
 module.exports = function(context) {
   return {
-    CallExpression: function(node) {
-      if (node.callee.type !== 'MemberExpression') return
-      if (node.callee.object.name !== '$') return
-
-      const name = node.callee.property.name
-      switch (name) {
-        case 'ajax':
-        case 'get':
-        case 'getJSON':
-        case 'getScript':
-        case 'post':
-          context.report({
-            node: node,
-            message: 'Prefer fetch to $.' + name
-          })
+    CallExpression: utils.withProperty(
+      ['ajax', 'get', 'getJSON', 'getScript', 'post'],
```

Comment:
You can also pass an array of property or method names into these util functions if you want to check multiple property/method names.